src/components/mainpage/chatscreen/components.mainpage.chatscreen.chatroom.css

```
@@ -0,0 +1,50 @@
+.chatroom {
+  display: flex;
+  flex-direction: column;
+  overflow: scroll;
+  padding-top: 8rem;
+  padding-bottom: 5rem;
+}
+
+img {
+  width: 4rem;
+  border-radius: 2rem;
+}
+
+.user1 {
+  display: flex;
+  padding-left: 2rem;
+  justify-content: left;
+  margin-bottom: 1.5rem;
+}
+.user1 p {
+  padding-top: 10px;
+  padding-right: 15px;
+  padding-left: 15px;
+  padding-top: 8px;
```

`padding-top`이 두 번 들어가 있네요!

```suggestion
  padding: 8px 15px 0 15px;
```

이렇게 간단하게 쓸 수도 있습니다~!